src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -451,7 +451,7 @@ public CompletableFuture<Void> runWithReplay(Path path, @Nullable Integer replay
         }))
         .thenRun(() -> {
           try {
-            Process processForReplay = forgedAllianceService.startReplay(path, replayId);
+            Process processForReplay = forgedAllianceService.startReplay(path, replayId, forgedAlliancePrefs.isAllowReplaysWhileInGame());
```

Comment:
Why pass the prefs value as a parameter? 

This is just the startReplay function so will only be used for replays so the isAllowReplaysWhileInGame should be entirely invisible to the caller.